test/comprehensions.coffee

```diff
@@ -9,6 +9,12 @@
 
 # TODO: refactor comprehension tests
 
+evens = ->
```

Comment:
Okay. Why make `evens` a function? Can’t it stay an assignment like it was before?
```coffee
evens = for num in [1, 2, 3, 4, 5, 6] when not (num & 1)
            num *= -1
            num -= 2
            num * -1
```